kubectl/README.md

```diff
@@ -0,0 +1,51 @@
+# Knative with knctl
```

Comment:
We should rename it to something like `kubectl` or `Deploy Kubernetes Resource with kubectl`?

Reply:
Ah, you catch it, my bad.